### DISCUSSION TITLE
MongoDB Integration problems

### DISCUSSION THREAD

**Discussion by aeghin** [vercel/community#4517] [Help]
### Summary

I just tried to deploy my first application on Vercel and I used the Integration with MongoDB Atlas for a seamless connection. I have came across a problem that I can't seem to fix. 

The application deploys just fine. It's when. I try to make any API calls where I get this error: 

Failed to load resource: net::ERR_CONNECTION_REFUSED

I know for sure it's gotta be something I've missed during the integration with MongoDB. 

Anyone familiar with this particular integration that can help me troubleshoot this problem?

### Example

https://track-x-five.vercel.app/

### Steps to Reproduce

Try to register and you'll have the error show in the console.

**Comment by amyegan** [comment]
Hi @aeghin! It looks like your deployed application is trying to hit `localhost` instead of the site domain. You should check the environment variables or anywhere that this may be hardcoded in your app.

<img width="498" alt="console log shows connection refused error on post to localhost 3001" src="https://github.com/vercel/community/assets/8247105/cf4b9715-182b-4535-a1da-af6386202940">
